crates/ty_project/src/lib.rs

```diff
@@ -385,14 +409,14 @@ impl Project {
         // both have a durability of `LOW`.
         if path.is_vendored_path() {
             return false;
+        } else if path.is_system_virtual_path() {
+            return true;
```

Comment:
It probably doesn't matter in practice but a virtual file could exist that isn't in `open_files`. That's why I think it's incorrect to move this check above `open_files`.